pkg/api/v1alpha1/cloudstackmachineconfig_types.go

```diff
@@ -56,13 +56,13 @@ type CloudStackResourceDiskOffering struct {
 	// +optional
 	CustomSize int64 `json:"customSizeInGB,omitempty"`
 	// path the filesystem will use to mount in VM
-	MountPath string `json:"mountPath"`
+	MountPath string `json:"mountPath,omitempty"`
```

Comment:
I think we'll need some preflight validation in addition to this to make sure the fields are set as expected. Previously, it was assumed that all these fields were present and that may now not be the case.

Reply:
I remembered that MountPath/device/fs/label is required fields if disk offering is configured.